alacritty_terminal/build.rs

```diff
@@ -0,0 +1,15 @@
+use std::env;
+use std::fs::File;
+use std::io::Write;
+use std::path::Path;
+
+fn main() {
+    let dest = env::var("OUT_DIR").unwrap();
+    let mut file = File::create(&Path::new(&dest).join("ansi_array.rs")).unwrap();
+
+    file.write_all(b"pub const U8_TO_STR: [&str; 256] = [").unwrap();
+    for i in 0..=u8::MAX {
+        write!(file, r#""{};","#, i).unwrap();
+    }
+    file.write_all(b"];").unwrap();
```

Comment:
No, this is not how to do this.

Reply:
How are you supposed to do it?

Reply:
You should use a fast but readable implementation.

Reply:
I changed it. Is it better now?

Reply:
Update:
This is no longer applicable right now but I still considered it to be solved by my update.